spk/tt-rss/Makefile

```diff
@@ -1,6 +1,6 @@
 SPK_NAME = tt-rss
-SPK_VERS = 17.4
-SPK_REV = 11
+SPK_VERS = 20200829
```

Comment:
Since tt-rss is no longer versioned I guessed the best option was to use the date of the commit.

Reply:
I agree